docs/src/getting_started/using_cmaize_as_build_system/cmake_like_project.rst

```diff
@@ -0,0 +1,174 @@
+*******************************************
+Example Project using CMake Style Structure
+*******************************************
+
+This chapter provides an overview of a project structured in a style that
+is common to CMake projects that uses CMaize as its build system.
+
+Project Overview
+================
+
+``MyCMakeProject`` is a simple C++ project that consists of the following
+components:
+
+* The ``TravelCostUtils`` library that contains some simple arithmetic
+  calculations for use in calculating travel costs.
+
+* The ``TravelCostCalc`` executable that calculates travel cost for a set of
+  predefined values using functions in ``TravelCostUtils``.
+
+* The ``Test`` directory contains some tests written using the Catch2 library
+  that test that ``TravelCostUtils`` works properly.
+
+Here is the file structure of the project:
+
+::
+
+    MyCMakeProject
```

Comment:
Generally speaking it's best to use snake_case names for files and directories. This will save you a lot of headache when dealing with multiple operating systems. In particular Mac (and technically Windows, even though it doesn't appear to) use case-insensitive names. The exception to this is when a file has a standard name that is case-sensitive (`CMakeLists.txt`, `README.md`, *etc.*).

Reply:
I updated the projects to use snake_case for files and directories.